testsuite/preview/expansion/src/test/java/org/wildfly/test/preview/microprofile/telemetry/AbstractOpenTelemetryIntegrationTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.wildfly.test.preview.microprofile.telemetry;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.core.Response;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.arquillian.testcontainers.api.Testcontainer;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.operations.common.Util;
+import org.jboss.as.test.shared.CdiUtils;
+import org.jboss.as.test.shared.observability.containers.OpenTelemetryCollectorContainer;
+import org.jboss.as.test.shared.observability.signals.jaeger.JaegerResponse;
+import org.jboss.dmr.ModelNode;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+
+import java.net.URL;
+import java.util.stream.Collectors;
+
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OUTCOME;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.RESULT;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUBSYSTEM;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUCCESS;
+import static org.jboss.as.test.shared.observability.setuptasks.ServiceNameSetupTask.SERVICE_NAME;
+
+public abstract class AbstractOpenTelemetryIntegrationTest {
```

Comment:
@jasondlee Please review.

Review in general of course, but I put the comment here because soonish these classes will be promoted to standard WildFly and end up in testsuite/integration/microprofile. So now is the time to think whether this is a good name for a base class that will end up in that module, are the packages what you'll want etc. IOW let's plan for promotion.

Reply:
@gaol There's already a base otel test class in ts/integ/mp. To make promoting these tests out of preview, I'd suggest, say, forking and extending that class. It looks like the only new functionality you're adding is the log retrieval, so if all else is the same, there will be much less to merge/migrate when that time comes.